addons/account/views/res_config_view.xml

```diff
@@ -98,7 +98,32 @@
                             </div>
                         </div>
                     </group>
-                    <group string="Accounting &amp; Finance">
+                    <group string="Accounting &amp; Finance" name="accounting">
+                        <label for="use_anglo_saxon"/>
+                        <div>
+                            <field name="use_anglo_saxon" class="oe_inline"/>
+                            <span title='This field is company-dependant, you are setting the config for this company only.'><i class='fa fa-building-o' groups="base.group_multi_company"/></span>
```

Comment:
This is very strange to replicate this ugly icon and the same text so many times...